mypy/main.py

```diff
@@ -612,6 +612,11 @@ def add_invertible_flag(flag: str,
     # --local-partial-types disallows partial types spanning module top level and a function
     # (implicitly defined in fine-grained incremental mode)
     parser.add_argument('--local-partial-types', action='store_true', help=argparse.SUPPRESS)
+    # --tweaked-deps adds some more dependencies that are not semantically needed, but
```

Comment:
Mention that this is an experimental, undocumented feature that may go away or be changed at any time.